webapp/channels/src/components/textbox/textbox.tsx

```diff
@@ -278,13 +278,17 @@ export default class Textbox extends React.PureComponent<Props> {
             textboxClassName += ' textarea--has-labels';
         }
 
+        if (this.props.hasError) {
+            textboxClassName += ' textarea--has-errors';
+        }
+
         return (
             <div
                 ref={this.wrapper}
                 className={classNames('textarea-wrapper', {'textarea-wrapper-preview': this.props.preview})}
             >
                 <div
-                    tabIndex={this.props.tabIndex || 0}
```

Comment:
in preview mode, this was causing the focus to be on the preview element. Now, it needs to be explicitly defined.